packages/contracts/src/plugins/governance/majority-voting/token/build-metadata.json

```diff
@@ -1,13 +1,105 @@
 {
   "ui": "",
-  "change": "",
+  "change": "- The `ProposalCreated` event is now emitted with the correct start and end date if 0 is used as an input parameter.\n- The ability to create a proposal now depends on the membership status of the current instead of the snapshot block.",
   "pluginSetupABI": {
-    "prepareInstallation": [
-      "tuple(uint8 votingMode, uint64 supportThreshold, uint64 minParticipation, uint64 minDuration, uint256 minProposerVotingPower) votingSettings",
-      "tuple(address addr, string name, string symbol) tokenSettings",
-      "tuple(address[] receivers, uint256[] amounts) mintSettings"
-    ],
-    "prepareUpdate": [""],
-    "prepareUninstallation": [""]
+    "prepareInstallation": {
+      "description": "The information required for the installation of build 2.",
```

Comment:
Same. It brings little meaning to add a message like this, just to fill the blank.

Reply:
See here: https://github.com/aragon/osx/pull/375/files#r1200096582